### PR TITLE
added get for id's method to all entity collections

### PR DESCRIPTION
**Pull request by peetjvv**
- this will speed up fetching entities for specific id's dramatically, especially shapes

### CODE REVIEW

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs [hunk 1]
```diff
@@ -167,6 +168,43 @@ public IEnumerable<Shape> Get(string entityId)
...
+                return new List<Shape>();
+            }
+            var sql = new StringBuilder("SELECT id, shape_pt_lat, shape_pt_lon, shape_pt_sequence, shape_dist_traveled FROM shape WHERE FEED_ID = :feed_id AND id = :id0");
+            var parameters = new List<NpgsqlParameter>();
+            parameters.Add(new NpgsqlParameter("feed_id", DbType.Int64));
+            parameters[0].Value = _id;
+            int i = 0;
+            foreach (var entityId in entityIds)
+            {
+                if (i > 0) sql.Append($" OR id = :id{i}");
```
Comment (timminata) [inline GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs:189]: Hopefully we never hit an issue with the SQL command being too long. Just keep it in mind though, could become an issue if you query with 1000's of ids.

**Review by timminata — APPROVED** [review]

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs (hunk 1 above)
Comment (peetjvv) [reply GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs:189]: I'll test it quickly

Comment (peetjvv) [reply GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs:189]: going to complete, this doesn't seem to be an issue

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs [hunk 2]
```diff
@@ -167,6 +168,43 @@ public IEnumerable<Shape> Get(string entityId)
...
+        /// </summary>
+        /// <param name="entityIds"></param>
+        /// <returns></returns>
+        public IEnumerable<Shape> Get(List<string> entityIds)
+        {
+            if (entityIds.Count() == 0)
+            {
+                return new List<Shape>();
+            }
+            var sql = new StringBuilder("SELECT id, shape_pt_lat, shape_pt_lon, shape_pt_sequence, shape_dist_traveled FROM shape WHERE FEED_ID = :feed_id AND id = :id0");
```
Comment (timminata) [inline GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs:182]: Order returned by PostgreSQL is not deterministic from what I can tell, you should add order by

Comment (peetjvv) [reply GTFS.DB.PostgreSQL/Collections/PostgreSQLShapesCollection.cs:182]: fixed in master

**Pull request merged**